### DISCUSSION TITLE
修改Recent Posts页面

### DISCUSSION THREAD

**Discussion by leonce28** [ppoffice/hexo-theme-icarus#871] [Q&A]
我想让recent post页面的缩略图能够随机获取，但是我没找到对应的源码。有什么建议吗？

**Comment by ppoffice** [comment]
@leoncelei 源码在这里：
https://github.com/ppoffice/hexo-component-inferno/blob/0.12.0/src/view/widget/recent_posts.jsx

修改方法参见：[挂件/评论插件/分享按钮...的布局文件在哪里？我如何个性化内置的挂件/评论插件/分享按钮...？](https://ppoffice.github.io/hexo-theme-icarus/uncategorized/%E5%B8%B8%E8%A7%81%E9%97%AE%E9%A2%98/#%E5%B8%83%E5%B1%80)
